doc/workers.rst

```diff
@@ -68,6 +68,66 @@ You can update an existing environment with the following::
 
       ~/ramp_deployment $ ramp setup update-conda-env --event-config events/iris_test/config.yml
 
+Event configuration
+^^^^^^^^^^^^^^^^^^^
+
+Create an event config.yml (see :ref:`deploy-ramp-event`) and update the
+'worker' section, which should look something like::
+
+      ramp:
+          problem_name: iris
+          event_name: iris_ramp_test
+          event_title: "iris ramp conda test"
```

Comment:
```suggestion
          event_title: "Iris classification challenge"
```

Reply:
I adapted event_title from the AWS section below which is:
event_title: "iris ramp aws test" just to be coherent with what follows, but I guess we can change both to be "Iris classification challenge"